content/swagger/resources/product_models/routes/product_models_code.yaml

```diff
@@ -31,7 +31,7 @@ patch:
     - "2.0"
     - "2.1"
     - "2.2"
-  description: This endpoint allows you to update a given product model. Know more about <a href="/documentation/update.html#update-behavior">Update behavior</a>. Note that if no product model exists for the given code, it creates it.
+  description: This endpoint allows you to update a given product model. Know more about <a href="/documentation/update.html#update-behavior">Update behavior</a>. Note that if no product model exists for the given code, it creates it. If you are using a v2.0 Enterprise Edition PIM, permissions based on your user groups are applied to the product model you try to update. It may result in the creation of a draft if you only have edit rights through the product model's categories.
```

Comment:
Remove this change, it's already covered in another PR.

Reply:
fix